### PR TITLE
Bump to 1.0.0rc5

### PR DESCRIPTION
**Pull request by VVsssssk**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the back-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs/en/changelog.md [hunk 1]
```diff
@@ -1,5 +1,29 @@
...
+
+#### Improvements
+
+- Fix docs link and add docs link checker (#1811)
+- Fix the cross-codebase reference problem in metafile README (#1644)
+- Update the cn docs about getting started (#1715)
+
+#### Bug Fixes
+
+- Fix PointRCNN produce Nan loss bug in training (#1874)
```
Comment (ZCMax) [inline docs/en/changelog.md:17]: Fix Nan loss bug in PointRCNN training

File: docs/en/changelog.md [hunk 2]
```diff
@@ -1,5 +1,29 @@
...
+- Fix segmentation visualization bug due to wrong parameter passing (#1858)
```
Comment (ZCMax) [inline docs/en/changelog.md:18]: point cloud segmentation visualization bug

File: docs/en/changelog.md [hunk 3]
```diff
@@ -1,5 +1,29 @@
 ## Changelog
 
+### v1.0.0rc5 (9/10/2022)
```
Comment (Tai-Wang) [inline docs/en/changelog.md:3]: 11/10/2022

File: docs/en/changelog.md [hunk 4]
```diff
@@ -1,5 +1,29 @@
...
+
+#### New Features
+
+- Add indoor monocular 3d object detector ImVoxelNet on SUN RGB-D (#1738)
```
Comment (Tai-Wang) [inline docs/en/changelog.md:7]: Support ImVoxelNet on SUN RGB-D

File: docs/en/changelog.md [hunk 5]
```diff
@@ -1,5 +1,29 @@
...
+
+#### Improvements
+
+- Fix docs link and add docs link checker (#1811)
+- Fix the cross-codebase reference problem in metafile README (#1644)
+- Update the cn docs about getting started (#1715)
```
Comment (Tai-Wang) [inline docs/en/changelog.md:13]: Chinese documentation

File: docs/en/changelog.md [hunk 6]
```diff
@@ -1,5 +1,29 @@
...
+
+#### Bug Fixes
+
+- Fix Nan loss bug during PointRCNN training (#1874)
+- Fix point cloud segmentation visualization bug due to wrong parameter passing (#1858)
+- Fix visualization bug due to empty prediction labels (#1725)
```
Comment (Tai-Wang) [inline docs/en/changelog.md:19]: Fix a visualization bug that is potentially triggered by empty prediction labels

**Review by Tai-Wang — APPROVED** [review]

**Pull request merged**